deluge/ui/gtkui/mainwindow.py

```diff
@@ -106,6 +105,7 @@ def patched_connect_signals(*a, **k):
         self.window = self.main_builder.get_object("main_window")
 
         self.window.set_icon(deluge.ui.gtkui.common.get_deluge_icon())
+        self.window.set_keep_above(False)
```

Comment:
in my testing the mainwindow kept going on top of the preferences window and maybe others...this could have been an issue with other things i was working on though... so ill try and test see what happens.